Gems/EMotionFX/Code/Source/Editor/SkeletonModel.cpp

```diff
@@ -234,8 +264,9 @@ namespace EMotionFX
         Node* node = static_cast<Node*>(index.internalPointer());
         AZ_Assert(node, "Expected valid node pointer.");
 
+        const bool isRootNode = node == m_characterRootNode;
         const size_t nodeIndex = node->GetNodeIndex();
-        const NodeInfo& nodeInfo = m_nodeInfos[nodeIndex];
+        const NodeInfo& nodeInfo = isRootNode ? m_nodeInfos[0] : m_nodeInfos[nodeIndex + 1];
```

Comment:
question: Is it worth sanity checking (assert?) that `nodeIndex + 1` is < `m_nodeInfos.size()`?